xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,888 @@
+     <link xlink:href="https://www.suse.com/documentation/sles-15/book_rmt/data/sec_rmt_client_clientsetupscript.html">get
+     updates from the internal RMT server</link>.
+    </para>
+   </sect3>
+   <sect3>
+    <title>Updating The Mirror</title>
+    <para>
+     Follow the procedure in
+     <link xlink:href="https://www.suse.com/documentation/sles-15/book_rmt/data/sec_rmt_mirroring_export_import.html" />.
+    </para>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.helm-charts">
+   <title>Helm Chart Repository Mirror</title>
+   <para>
+    Helm Charts provide a simple way to deploy entire application stacks
+    including all their dependencies and configurations. The repository consists
+    of chart packages that contain the necessary information and an index file.
+   </para>
+   <para>
+    The helm charts will require images available from a registry mirror. The
+    following steps will aid you in mirroring all images required by the &suse;
+    provided helm charts. To make use of the helm charts, you must complete
+    <xref linkend="sec.deploy.scenarios.airgap.container-registry"/>.
+   </para>
+   <sect3>
+    <title>Mirror Configuration</title>
+    <para>
+     Set up an internal Helm chart mirror. This can be any webserver capable of
+     hosting static files. We recommend providing the webserver on your internal
+     container image registry mirror.
+    </para>
+    <para>
+     You do not need a Helm chart repository mirror on the external
+     network. You will perform the steps required for pulling and mirroring
+     charts and container images on your external container image registry
+     mirror.
+    </para>
+    <para>
+     You must have a working container image registry mirror in place inside
+     the air gapped network.
+    </para>
+   </sect3>
+   <sect3>
+    <title>Client Configuration</title>
+    <para>
+     Add the webserver as a repo to <command>helm</command> on the admin node.
+     </para>
+<screen>&prompt.user;<command>helm repo add SUSE-MIRROR <replaceable>&lt;internal-mirror.hostname&gt;</replaceable></command>
+   </screen>
+   </sect3>
+   <sect3 xml:id="sec.deploy.scenarios.airgap.helm-charts.update">
+    <title>Updating The Mirror</title>
+    <para>
+     To update the repository, you simply use <command>helm mirror</command> to
+     refresh the chart definitions on the server.
+    </para>
+    <procedure>
+     <step>
+      <para>
+      Install the
+      <link xlink:href="https://github.com/openSUSE/helm-mirror">helm-mirror</link>
+      plugin from the <filename>Containers</filename> module.
+     </para>
+<screen>&prompt.sudo;<command>zypper in helm-mirror</command>
+      </screen>
+      </step>
+      <step>
+       <para>
+      Download all charts from the repository.
+     </para>
+<screen>&prompt.user;<command>helm mirror https://kubernetes-charts.suse.com <replaceable>/charts/</replaceable></command>
+      </screen>
```

Comment:
@flavio @r0ckarong it's not, we have to provide support for windows too so they can merge that.

Reply:
> @flavio @r0ckarong it's not, 

Ok, we have to update the instructions to invoke `helm-mirror` instead of what is currently written.

> we have to provide support for windows too so they can merge that.

Do you mean we have to provide some code that handles system wide plugins on windows?

Reply:
Yes, that is what they told me, that we must provide a way to support all OS that Helm works on, so I need to put some time into it to do it.